src/commands/move.ts

```diff
@@ -0,0 +1,144 @@
+import { Command, Flags } from '@oclif/core';
+import { ConfigService } from '../services/config.service';
+import { DriveFolderService } from '../services/drive/drive-folder.service';
+import { CLIUtils } from '../utils/cli.utils';
+import {
+  ItemNotFoundError,
+  MissingCredentialsError,
+  NotValidFolderUuidError,
+  NotValidItemUuidError,
+} from '../types/command.types';
+import { ValidationService } from '../services/validation.service';
+import { DriveFileService } from '../services/drive/drive-file.service';
+import { DriveFileItem, DriveFolderItem } from '../types/drive.types';
+import { ErrorUtils } from '../utils/errors.utils';
+
+export default class Move extends Command {
+  static readonly args = {};
+  static readonly description = 'Move a folder/file into a destination folder.';
+
+  static readonly examples = ['<%= config.bin %> <%= command.id %>'];
+
+  static readonly flags = {
+    ...CLIUtils.CommonFlags,
+    item: Flags.string({
+      char: 'i',
+      description: 'The item id to be moved (it can be a file id or a folder id).',
+      required: false,
+    }),
+    destination: Flags.string({
+      char: 'd',
+      description: 'The destination folder id where the item is going to be moved.',
+      required: false,
+    }),
+  };
+
+  public async run() {
+    const { flags } = await this.parse(Move);
+
+    const nonInteractive = flags['non-interactive'];
+
+    const userCredentials = await ConfigService.instance.readUser();
+    if (!userCredentials) throw new MissingCredentialsError();
+
+    const itemUuid = await this.getItemUuid(flags['item'], nonInteractive);
```

Comment:
should we use `id` or `originId` here to be consistent? we don't use the flag `item` anywhere else